sql/driver/driver_test.go

```diff
@@ -31,7 +32,7 @@ var isError = testutils.IsError
 
 func setup(t *testing.T) (*server.TestServer, *sql.DB) {
 	s := server.StartTestServer(nil)
-	db, err := sql.Open("cockroach", "https://root@"+s.ServingAddr()+"?certs=test_certs")
+	db, err := sql.Open("cockroach", "https://"+security.RootUser+"@"+s.ServingAddr()+"?certs=test_certs")
```

Comment:
we're not quite there yet. I'd first like to get root and node to be used in their proper places (it's still a bit messy). After that, most tests should move to a newly-created user for non-internal and non-admin operations.